app/src/main/java/com/google/samples/apps/sunflower/adapters/GardenPlantingAdapter.kt

```diff
@@ -88,4 +112,16 @@ private class GardenPlantDiffCallback : DiffUtil.ItemCallback<PlantAndGardenPlan
     ): Boolean {
         return oldItem.plant == newItem.plant
     }
-}
\ No newline at end of file
+}
+
+class GardenPlantingDetailsLookup(private val recyclerView: RecyclerView) :
```

Comment:
nit: line wrapping for class declaration

Reply:
Done.